gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -114,6 +116,7 @@ public static Builder builder() {
         .setWriteChannelOptions(AsyncWriteChannelOptions.DEFAULT)
         .setRequesterPaysOptions(RequesterPaysOptions.DEFAULT)
         .setCooperativeLockingOptions(CooperativeLockingOptions.DEFAULT)
+        .setEnableDebugLogging(ENABLE_DEBUG_LOGGING_DEFAULT)
```

Comment:
Why is this required? Not set could be simply "disabled".

Reply:
Removed